common/config/rush/pnpm-lock.yaml

```diff
@@ -6355,6 +6355,12 @@ packages:
     dev: false
     resolution:
       integrity: sha512-30wUB3fLXnMT8FDksNtoeHbg85GbT0qdei9FOn5eb/dqFOtuJu2oze1kf4YQkDvj/lZZAAx6Ol0NEwEiIBeIeg==
+  /rhea/1.0.21:
```

Comment:
Can we pretty please have the rhea update in a separate PR.... ?

Reply:
I can, but we should merge that first then (I actually ran into the idle error once when running the tests that the latest version fixes.)

Reply:
Done. See #8757

Reply:
#8757 is merged and this branch has been updated. But the lock file still has some sha changes...